main.go

```diff
@@ -12,6 +12,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
```

Comment:
this import move doesn't look right.

Reply:
What do you mean by "move"? As in, this should stay in `main.go` instead of in `http_server.go` (where it was moved to)?